rpcapd/rpcapd.c

```diff
@@ -194,7 +198,13 @@ int main(int argc, char *argv[])
 	mainhints.ai_socktype = SOCK_STREAM;
 
 	// Getting the proper command line options
-	while ((retval = getopt(argc, argv, "b:dhip:4l:na:s:f:v")) != -1)
```

Comment:
Or

    #ifdef HAVE_OPENSSL
        #define SSL_OPTS  "SK:C:"
    #else
        #define SSL_CLOPTS
    #endif

    #define CLOPTS "b:dhip:4l:na:s:f:v" SSL_CLOPTS

and then use `CLOPTS` as the third argument to `getopt()`.

Reply:
Switch to [optparse()](https://github.com/skeeto/optparse) instead? One single re-entrant file.

Reply:
That might be worth considering, but it's orthogonal to adding SSL and to the various ways of making the availability of SSL-specific flags conditional.